### PR TITLE
improve button css

### PR DESCRIPTION
**Pull request by timoschilling**

### CODE REVIEW

File: fs_src/style.css [hunk 1]
```diff
@@ -25,6 +25,12 @@ button {
   border-radius: 18px;
   border: none;
   cursor: pointer;
+  background: #0071e3;
+}
+
+button label {
+  position: relative;
+  cursor: pointer;
```
Comment (timoschilling) [inline fs_src/style.css:33]: Without this `cursor: pointer;` the mouse pointer was a hand on the button but a arrow on the button label

File: fs_src/index.html [hunk 2]
```diff
@@ -43,7 +43,7 @@ <h1 class="">General Settings</h1>
       </div>
       <div class="form-control">
         <label></label>
-        <button class="btn" id="sys_save_btn">
```
Comment (timoschilling) [inline fs_src/index.html:46]: we do element styling now, so no need for class attachments

**Review by rojer — APPROVED** [review]
LGTM, please rebase

**Comment by timoschilling** [comment]
rebased

**Pull request merged**